oak-run-commons/src/main/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/FlatFileNodeStoreBuilder.java

```diff
@@ -130,7 +131,7 @@ public class FlatFileNodeStoreBuilder {
     private Predicate<String> pathPredicate = path -> true;
 
     private final boolean compressionEnabled = Boolean.parseBoolean(System.getProperty(OAK_INDEXER_USE_ZIP, "true"));
-    private final boolean useLZ4 = Boolean.parseBoolean(System.getProperty(OAK_INDEXER_USE_LZ4, "false"));
+    private final boolean useLZ4 = Boolean.parseBoolean(System.getProperty(OAK_INDEXER_USE_LZ4, DEFAULT_OAK_INDEXER_USE_LZ4));
     private final Compression algorithm = compressionEnabled ? (useLZ4 ? new LZ4Compression() : Compression.GZIP) :
```

Comment:
Should we move the logic to read the system prop for compression to some util class - maybe IndexStoreUtils - assuming this would also be needed for the TreeFS builder later on ?

Reply:
Note that we have https://www.javadoc.io/doc/org.apache.jackrabbit/oak-commons/latest/org/apache/jackrabbit/oak/commons/properties/SystemPropertySupplier.html

Reply:
> Should we move the logic to read the system prop for compression to some util class - maybe IndexStoreUtils - assuming this would also be needed for the TreeFS builder later on ?

I created static methods in IndexStoreUtils to read the compression configuration, the logic is all there.

Reply:
> Note that we have https://www.javadoc.io/doc/org.apache.jackrabbit/oak-commons/latest/org/apache/jackrabbit/oak/commons/properties/SystemPropertySupplier.html

I was not aware of this class, it's a useful class. I will use it in future but I'd rather not introduce more changes in this PR.